example/workflows/browseExample.yml

```diff
@@ -162,6 +162,7 @@ CookbookBrowseExample:
         bucket: '{$.meta.buckets.internal.name}'
         buckets: '{$.meta.buckets}'
         distribution_endpoint: '{$.meta.distribution_endpoint}'
+        cmr: '{$.meta.cmr}'
```

Comment:
Why does the MoveGranuleStep need `cmr` configuration information?

Reply:
removed